.travis.yml

```diff
@@ -13,6 +13,11 @@ scala:
   - 2.10.7
   - 2.11.12
   - 2.12.5
+  - 2.13.0-M3
+
+matrix:
+  allow_failures:
```

Comment:
It might be helpful to leave a comment here about why this is happening.

Reply:
no problem, I'll add a comment